### PR TITLE
chore: USMCA COO example added

### PR DESCRIPTION
**Pull request by nissimsan**
https://github.com/w3c-ccg/traceability-vocab/issues/238

### CODE REVIEW

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml [hunk 1]
```diff
@@ -128,5 +128,101 @@ required:
   - certifier
 example: |-
   {
-    "issue": "https://github.com/w3c-ccg/traceability-vocab/issues/238"
+    "id": "http://example.org/credentials/",
```
Comment (OR13) [inline docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:131]: id is sad

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml [hunk 2]
```diff
@@ -128,5 +128,101 @@ required:
...
+            }
+          },
+          "originCriterion": "A",
+          "countryOfOrigin": "MX"
+        }
+      ],
+      "blanketPeriodFrom": "2021-06-22",
+      "blanketPeriodTo": "2022-06-21"
+    },
+    "@context": [
```
Comment (OR13) [inline docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:217]: prefer context at top

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml [hunk 3]
```diff
@@ -128,5 +128,101 @@ required:
...
+          "email": "Woodrow_Hackett@example.net",
+          "phoneNumber": "555-716-2400",
+          "faxNumber": "555-853-9640"
+        }
+      ],
+      "producerDetails": [
+        {
+          "type": "Organization",
+          "name": "Terry Inc",
+          "description": "Optimized discrete core",
```
Comment (OR13) [inline docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:178]: best description ever.

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml [hunk 4]
```diff
@@ -128,5 +128,101 @@ required:
...
+    ],
+    "issuanceDate": "2021-02-04T20:29:37+00:00",
+    "issuer": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+    "credentialSubject": {
+      "type": "USMCACertificateOfOrigin",
+      "certifier": "Importer",
+      "exporterDetails": {
+        "type": "Organization",
+        "name": "Torphy, Fahey and Stamm",
+        "description": "Synergized composite data-warehouse",
```
Comment (OR13) [inline docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:143]: ideally these would be more realistic.

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml [hunk 5]
```diff
@@ -128,5 +128,101 @@ required:
...
   {
-    "issue": "https://github.com/w3c-ccg/traceability-vocab/issues/238"
+    "id": "http://example.org/credentials/",
+    "type": [
+      "VerifiableCredential"
+    ],
+    "issuanceDate": "2021-02-04T20:29:37+00:00",
+    "issuer": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+    "credentialSubject": {
+      "type": "USMCACertificateOfOrigin",
```
Comment (OR13) [inline docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:138]: should this type be actually above?

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml [hunk 6]
```diff
@@ -128,5 +128,101 @@ required:
   - certifier
 example: |-
   {
-    "issue": "https://github.com/w3c-ccg/traceability-vocab/issues/238"
+    "id": "http://example.org/credentials/",
+    "type": [
+      "VerifiableCredential"
```
Comment (OR13) [inline docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:133]: credential type ambiguous

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml (hunk 1 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:131]: Right. Very sad.

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml (hunk 2 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:217]: Agreed. Fixing...

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml (hunk 5 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:138]: Yes, true. I iterated a bit on this structure. This should have been moved back up. Fixing.

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml (hunk 6 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:133]: Fixing.

File: docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml (hunk 4 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/USMCACertificateOfOrigin.yml:143]: I don't disagree. I don't know if I'll have time to generate nice test data by today, though, so might fix this later. 
It's the kind of generator-data we used to have all over the place, so not worse that what we're used to. :)

**Review by mprorock — CHANGES_REQUESTED** [review]
needs an `id`

**Comment by OR13** [comment]
wonder why ci is failing.

**Comment by mkhraisha** [comment]
looks like a number of tests aren't passing here rather than a pipeline issue.

**Comment by OR13** [comment]
Looks like the examples are not valid VCs, use https://api.did.actor/issue or some other tool to make the examples valid

**Comment by nissimsan** [comment]
I see - we're actually verifying our verifiable credentials now! Brave new world...!
https://api.did.actor/issue is super helpful, @OR13 - thanks.

**Pull request closed without merging**